src/app/screens/SignUp/components/CustomInput/styles.module.scss

```diff
@@ -0,0 +1,12 @@
+@import '../../../../../scss/variables/colors';
+
+.custom-inputs {
+  border-radius: 5px;
+  height: 32px;
+  margin: 8px 23px 0 25px;
```

Comment:
i think margins wouldnt go inside input class, those spacing aren't a standard for all the app. you should send those as props, if shared on login, put them on login and send the className to the input